project/frontend/src/index.js

```diff
@@ -1,19 +1,22 @@
-import './index.css';
+import "./index.css"
 
-import React from 'react';
-import ReactDOM from 'react-dom';
+import React from "react"
+import ReactDOM from "react-dom"
+import {BrowserRouter} from "react-router-dom";
 
-import App from './App';
-import reportWebVitals from './reportWebVitals';
+import App from "./App"
+import reportWebVitals from "./reportWebVitals"
```

Comment:
sometimes a semicolon, sometimes not, do they not matter?

Reply:
Sorry this was in between the linter swap, will fix in next PR!